test/e2e/suite_test.go

```diff
@@ -51,78 +51,81 @@ var _ = BeforeSuite(func() {
 	framework.HandleFlags()
 	framework.AfterReadingAllFlags(&framework.TestContext)
 
-	creds, err := credentials.CreateAzureCredentialFile(false)
-	Expect(err).NotTo(HaveOccurred())
-	azureClient, err := azure.GetAzureClient(creds.Cloud, creds.SubscriptionID, creds.AADClientID, creds.TenantID, creds.AADClientSecret)
-	Expect(err).NotTo(HaveOccurred())
-	_, err = azureClient.EnsureResourceGroup(context.Background(), creds.ResourceGroup, creds.Location, nil)
-	Expect(err).NotTo(HaveOccurred())
-
-	// Need to login to ACR using SP credential if we are running in Prow so we can push test images.
-	// If running locally, user should run 'docker login' before running E2E tests
 	if testutil.IsRunningInProw() {
```

Comment:
I don't think we need to build & install CSI driver when we are testing in-tree volume plugin. Skipping this can decrease test duration by a fair amount.
```suggestion
	if testutil.IsRunningInProw() && testutil.IsTestingCSIDriver() {
```